.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: ruby
+sudo: required
+cache: bundler
+rvm:
+- 2.4.5
+services:
+- docker
+addons:
+  postgresql: '9.5'
+env:
+  global:
+  - DATABASE_USERNAME=root
+  - DATABASE_PASSWORD=smartvm
```

Comment:
I did it this way to be consistent with service_portal-api, but @bdunne is doing is differently here: https://github.com/ManageIQ/topological_inventory-core/blob/master/.travis.yml#L13  Let's pick a way.  cc @syncrou

Reply:
I think we went that way in the other repo because we're not using `database.yml`, but here you are also avoiding duplicating the user and password in the `psql` command ... 

If we're using `database.yml` here then this looks better. And if we apply this to the core side then we can DRY the user and password.

Reply:
Right, having a committed database.yml that's driven by the username and password env vars is what the service_portal-api does.

Reply:
What I don't like about DATABASE_URL is it overrides *all* values in database.yml including the test database.  It's not DRY with respect to database.yml in that case.  As such, local test is then different from travis test, albeit slightly

Reply:
> too much craziness in database.yml

Which craziness?  I don't know what you mean by that.

Reply:
I guess I don't know what an alternative to `DATABASE_URL` looks like that allows for a non-dirty git tree.

Reply:
See https://github.com/ManageIQ/service_portal-api/blob/master/config/database.yml#L5-L6

Reply:
Note those lines can be streamlined more with

```yaml
  username: <%= ENV.fetch('SERVICE_CATALOG_DATABASE_USERNAME', 'root') %>
  password: <%= ENV.fetch('SERVICE_CATALOG_DATABASE_PASSWORD', 'smartvm') %>
```

Reply:
Discussed with @syncrou @bdunne and @carbonin in person, and we are going to go the route of

- not commiting database.yml
- committing a database.dev.yml with basic defaults
- changing travis.yml to just copy database.dev.yml and/or try to use travis' built-in user/database so we can avoid the psql line